core/capabilities/integration_tests/mock_trigger.go

```diff
@@ -0,0 +1,113 @@
+package integration_tests
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/datastreams"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+const triggerID = "streams-trigger@1.0.0"
+
+type reportsSink struct {
+	triggers []streamsTrigger
+}
+
+func (r *reportsSink) sendReports(reportList []*datastreams.FeedReport) {
+	for _, trigger := range r.triggers {
+		resp, err := wrapReports(reportList, "1", 12, datastreams.SignersMetadata{})
```

Comment:
Should the signers metadata include signatures? Otherwise, how are we validating the reports further down the line?